packages/carbon-graphs/tests/unit/controls/Graph/GraphAxes-spec.js

```diff
@@ -1212,6 +1214,70 @@ describe('Graph - Axes', () => {
       expect(yTicks).toEqual(expectedYTicks);
       expect(y2Ticks).toEqual(expectedY2Ticks);
     });
+
+    it('tesing filter method for numeric', () => {
+      const localeAxisObj = utils.deepClone(axisTicks);
+      localeAxisObj.allowCallibration = false;
+      localeAxisObj.x = {
+        label: 'x axis',
+        lowerLimit: 2,
+        upperLimit: 8,
+      };
+      localeAxisObj.y = {
+        label: 'y axis',
+        lowerLimit: 0,
+        upperLimit: 30,
+      };
+      localeAxisObj.y2 = {
+        show: true,
+        label: 'y2 axis',
+        lowerLimit: 0,
+        upperLimit: 120,
+      };
+      graph = new Graph({ ...getAxes(localeAxisObj) });
+
+      const allXAxisElements = document.querySelectorAll(`.${styles.axisX}`);
+      const xTicks = [];
+      const expectedXTicks = [2, 3, 4, 5, 6, 7, 8];
+
+      for (let i = 1; i < allXAxisElements[0].childNodes.length; i += 1) {
+        xTicks.push(parseInt(allXAxisElements[0].childNodes[i].querySelector('text').textContent, 10));
+      }
```

Comment:
We can use the [map function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) for an array:
```suggestion
const xTicks = allXAxisElements[0].childNodes.map((node)=>parseInt(node.querySelector('text').textContent,10));
```